src/m_menu.c

```diff
@@ -2412,6 +2412,8 @@ static void M_DrawSetting(setup_menu_t *s, int accum_y)
 
       if (strings)
         strcpy(menu_buffer, strings[value]);
+      else if (flags & S_THRM_PCT)
+        M_snprintf(menu_buffer, 4, "%d%%", value);
```

Comment:
`%` is cut off for `100%`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1496/commits/44f9e9822c427f2199d154d4fa6ec6fdcb152767